rust/impl/iter/iter.rs

```diff
@@ -47,7 +47,7 @@ pub mod exposed
   };
 
   #[ doc( inline ) ]
-  pub use std::iter::zip;
```

Comment:
It is better to keep this code unchanged. 

The idea of the module components is: 
- to have only core functionality of the module, use no features 
- to have some extended functionality of the module, the feature `use_std` should be enabled 
- to have advanced functionality, use more features 

Please, analyze code and add `#[ cfg( feature = "use_std" ) ]`